### ISSUE TITLE
Monsters: Common used and Color pallets. 

### ISSUE THREAD

**Issue by gupgamingtv** [Bouncyrock/TaleSpire-Beta-AssetRequests#49]
I feel like there needs to be more baddies. Things pretty common such as an Owlbear, Bulette, Snake people (Yuan-Ti), and the like. When water tiles are added (its the most up-voted thing here) You will need shark/octopus/kraken/turtle. 

Some key creatures of D&D: Beholder, Mind Flayer, and at least 1 generic mini of the underdark race like Duergar and Drow.  A Purple Worm.

EASY TO ADD IDEA here: reskin what you have. You have a red dragon mini, change the color to blue, bronze, green, white, black. You can use the exact same mini, just change the color or make minor tweaks if you want but that would add some diversity with little effort and you can always go back later and update models.

Reskin/color template Dragons, wolf (multi use for hellhound, wolf, displacer beast, blink dog) Slime (black, ocher, green, blue) Hag (blue, green, dark).  

I feel the reskins will help a lot and be super easy to do, while the others may be a bit more work but would add a lot.

![owlbear](https://user-images.githubusercontent.com/65186863/81598828-85a3d580-938d-11ea-8a2b-68277b584643.jpg)
![beholder](https://user-images.githubusercontent.com/65186863/81598844-89375c80-938d-11ea-8b80-59ab635587bc.jpg)
![drow](https://user-images.githubusercontent.com/65186863/81598850-8b012000-938d-11ea-9a9b-b4c8f581855a.png)
![duergar](https://user-images.githubusercontent.com/65186863/81598856-8c324d00-938d-11ea-8799-1313500b10e3.jpg)
![bulette](https://user-images.githubusercontent.com/65186863/81598931-ad933900-938d-11ea-93fa-463754da78f8.jpg)

If anyone else has ideas along these lines or recommendations let me know! Thumbs up if you like the idea!

**Comment by LuPro** [comment]
Please keep in mind that the votes here are just for the devs to gauge interest, it is not a guarantee that high voted things will come into the game (for water tiles it is true they planned for them already, but I just want to point this out again).
Regarding the minis, thanks for the requests, definitely a nice list though keep in mind that talespire is not made just for dnd, so some of the suggestions might not come exactly like this and more in a "generalized version" when/if they do come.

regarding tinting: the devs plan to make minis tintable by players so that should fix that :)

**Comment by gupgamingtv** [comment]
Thanks for the comment,
Yea I believe I read water tiles were going to be in the works at some point! Everything put out so far is really nice, my group and I look forward to seeing all the future content. I do get tunnel vision about D&D so it is a good point to note that it's not exclusive to D&D. I also read somewhere that there would possibly be an uploadable option for stl files or something like that, although I cannot remember where I read it or if it was true or not. (My inner self wants to say YES!) lol

**Comment by LuPro** [comment]
Not directly stl files, but they do plan modding which can take any file format unity can take to add to the game (and eg: blender can convert STL to such formats). I'd recommend checking on the discord for this though, it's not 100% straightforward and that has been discussed there a lot already :)

**Comment by gupgamingtv** [comment]
I can only see a few chans in the discord, not sure if there are more but I just noticed the react message to get a role lol so i'll do that :D thanks

**Comment by LuPro** [comment]
Yeah, you'll have to get the taleweaver role to get access to the rest of the channels

**Comment by mefgengus** [comment]
Would love to see a dragon in flight,wings wide open,like the bats
